src/site/layouts/demo.js

```diff
@@ -7,7 +7,6 @@ export default ({ children }) => (
   <main>
     <Helmet>
       <title>PF Next</title>
-      <html lang="en-US" />
```

Comment:
why was this removed?

Reply:
Just to illustrate, in this PR, what it looks like when the build fails. I'm going to "fix" it in just a few, but wanted to get some eyes on the unhappy path first since it's what new contributors will be seeing for the first time.

Reply:
HAHA. I was like "what are you doin! It is going to fail!" Good idea for the illustration.